python/MyriaPythonWorker.py

```diff
@@ -0,0 +1,215 @@
+from __future__ import print_function
+import os
+import sys
+import socket
+import traceback
+import struct
+import cPickle
+import base64
+
+
+class SpecialLengths(object):
+    PYTHON_EXCEPTION_THROWN = -3
+    END_OF_STREAM = -4
+    NULL = -5
+
+
+class DataType(object):
```

Comment:
Why isn't this `enum` `0`-based (for compatibility with Java `enum`s)? Do you explicitly start the corresponding Java `enum` at `1`?